dynamodb/utils.js

```diff
@@ -14,7 +14,7 @@ var absPath = function (p) {
 
 var removeDir = function (relPath, callback) {
     var path = absPath(relPath);
-    rmdir(path, callback);
+    exec('rm -r ' + path, callback);
```

Comment:
Just wanted to check if this will work in Windows.

Reply:
@rehrumesh thanks for this question. It almost certainly does not work on Windows and I have no way to test it anyways. Do you have any ideas for how to address this?

Reply:
Take a look at this gist I found today: https://gist.github.com/laazebislam/845e61a18541255c0a29eeeb4e0141f4
It looks like it would work and remove a dependency.

Reply:
How about using [rimraf](https://www.npmjs.com/package/)?

Reply:
Adding the correct link for [rimraf](https://www.npmjs.com/package/rimraf)

I did look at that too but thought it overkill of what is needed when the code in the gist does the same thing. However, if rimraf is less risky then go ahead with that. We just need a fix.